resources/istio-gateway/values.yaml

```diff
@@ -0,0 +1,22 @@
+global:
+  istio:
+    gateway:
+      name: "compass-istio-gateway"
+      namespace: "compass-system"
+    proxy:
+      port: 15020
+
+  connector:
+    # If secrets do not exist they will be created
+    secrets:
+      rootCA:
+        name: kyma-gateway-certs
+        namespace: istio-system # For Ingress Gateway to work properly the namespace needs to be istio-system
```

Comment:
> If secrets do not exist they will be created

They won't be created as there is not certs-setup-job. I am also not sure if mTLS Gateway should be created as a part of KCP as I do not think any component uses it (without the connector it does not really make sense).